lib/httpClient.ts

```diff
@@ -4,13 +4,15 @@ import { Signer, Wallet } from 'ethers'
 import { JsonRpcProvider } from 'ethers/providers'
 import { arrayify, keccak256 } from 'ethers/utils'
 import { Claim, IRole } from './LoginStrategy.types'
+import { Policy } from 'cockatiel'
 
 export class CacheServerClient {
```

Comment:
Would it be possible to rename this file to be "CacheServerClient.ts" to match the name of the class?

Reply:
👍